https/docker-compose.yml

```diff
@@ -1,82 +1,76 @@
-db:
-  image: postgres
-  environment:
-   - POSTGRES_HOST_AUTH_METHOD=trust
+version: "3.7"
 
-uwsgi:
-  restart: always
-  image: quay.io/vanessa/sregistry
-  env_file:
-    - ./.minio-env
-  volumes:
-    - .:/code
-    - ./static:/var/www/static
-    - ./images:/var/www/images
-    # uncomment for PAM auth
-    #- /etc/passwd:/etc/passwd 
-    #- /etc/shadow:/etc/shadow
-  links:
-    - minio
-    - redis
-    - db
+services:
 
-nginx:
-  restart: always
-  image: quay.io/vanessa/sregistry_nginx
-  ports:
-    - "80:80"
-    - "443:443"
-  volumes:
-    - ./nginx.conf:/etc/nginx/conf.d/default.conf:ro
-    - ./uwsgi_params.par:/etc/nginx/uwsgi_params.par:ro
-    - /etc/ssl/certs:/etc/ssl/certs:ro
-    - /etc/ssl/private:/etc/ssl/private:ro
-  volumes_from:
-    - uwsgi
-  links:
-    - uwsgi
-    - db
+  db:
+    image: postgres:latest
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
 
-redis:
-  restart: always
-  image: redis:latest
+  uwsgi:
+    restart: always
+    image: quay.io/vanessa/sregistry
+    env_file:
+      - ./.minio-env
+    volumes:     
+      - .:/code
+      - ./images:/var/www/images
+      - ./static:/var/www/static
 
-scheduler:
-  image: quay.io/vanessa/sregistry
-  command: python /code/manage.py rqscheduler
-  volumes:
-    - .:/code
-  volumes_from:
-    - uwsgi
-  env_file:
-    - ./.minio-env
-  links:
```

Comment:
So the newer version doesn't need to define links? And `volumes_from` no longer exists so each needs to have all volumes listed explicitly?